specs/UrbanMobility/ArrivalEstimation/doc/spec.md

```diff
@@ -0,0 +1,67 @@
+# ArrivalEstimation
+
+## Description
+
+This Entity Type captures the estimation of arrival of a public transport vehicle, performing service on a route, to a certain stop. 
```

Comment:
The description could be clearer, something like:

> This Entity Type captures the estimated arrival time of a public transport vehicle reaching a particular stop, whilst the vehicle is servicing a particular route.

Reply:
good proposal